samples/dashboard/demos/climate/demo.js

```diff
@@ -424,37 +447,70 @@ function buildDates() {
 function buildDateTicks() {
     const dates = [];
 
-    for (let date = new Date(Date.UTC(2010, 0, 15)),
+    for (let date = new Date(Date.UTC(1951, 0, 15)),
```

Comment:
Do we want such old data?

Reply:
Yes, we want them all. :)

Reply:
It is climate data so we need a couple of centuries.